tests/test_marc.py

```diff
@@ -85,7 +85,7 @@ def test_line_to_bin_unicode(self):
                 marc = marcs.next()
                 self.assertTrue(marc.author() == \
                                 u'ƀƁƂƃƄƅƆƇƈƉƊƋƌƍƎƏƐƑƒƓƔƕƖƘƙƚƛƜƝƞƟƠơ '\
-                                '1900-1980 Verfasser$0(DE-588)118536389$4aut',
+                                '1900-1980 Verfasser (DE-588)118536389 aut',
                                 "Line MARC title didn't match pymarc title")
```

Comment:
Good suggestion @hornc, keeping branches specific to the feature makes it easier to navigate history. I can do a better job of following that example as well and using this as criteria for future reviews.